{"name": "test_login_with_invalidemail_and_validpassword_crenditial", "status": "broken", "statusDetails": {"message": "AttributeError: 'LoginPage' object has no attribute 'retrived_warning_message'. Did you mean: 'retrieve_warning_message'?", "trace": "self = <test.test_login.TestLogin object at 0x000001CBE77DFCE0>\n\n    def test_login_with_invalidemail_and_validpassword_crenditial(self):\n        home_page = HomePage(self.driver)\n        home_page.click_on_my_account_drop_menu()\n        login_page =home_page.select_login_option()\n        login_page.enter_email_address(\"swapnild0211@gmail.com\")\n        login_page.enter_password(\"1234\")\n        login_page.click_on_login_button()\n        expected_text=\"Warning: No match for E-Mail Address and/or Password.\"\n>       assert expected_text.__eq__(login_page.retrived_warning_message())\nE       AttributeError: 'LoginPage' object has no attribute 'retrived_warning_message'. Did you mean: 'retrieve_warning_message'?\n\ntest\\test_login.py:37: AttributeError"}, "attachments": [{"name": "stdout", "source": "02fb754b-fadf-495b-aaf0-a308e51f5bc8-attachment.txt", "type": "text/plain"}], "start": 1733160303940, "stop": 1733160308599, "uuid": "dd243417-8a24-4701-bef1-809d1313d3fb", "historyId": "c371f3039a48928ba234fc468508ee52", "testCaseId": "c371f3039a48928ba234fc468508ee52", "fullName": "test.test_login.TestLogin#test_login_with_invalidemail_and_validpassword_crenditial", "labels": [{"name": "tag", "value": "@pytest.mark.usefixtures('setup_and_teardown', 'log_on_failure')"}, {"name": "parentSuite", "value": "test"}, {"name": "suite", "value": "test_login"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "Agni"}, {"name": "thread", "value": "7036-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test.test_login"}]}